src/SFA.DAS.AssessorService.Database/Tables/Apply.sql

```diff
@@ -16,7 +16,8 @@ CREATE TABLE [Apply](
 	[DeletedBy] [nvarchar](256) NULL,
     [StandardApplicationType]  NVARCHAR(60) NULL ,  
     [StandardReference] NVARCHAR(10) NULL ,    
- CONSTRAINT [PK_Apply] PRIMARY KEY CLUSTERED 
+	[ApplyViaOptIn] BIT NULL DEFAULT 0, 
```

Comment:
If we've added a new field, and this is being used to determine if they applied via opt in to help with the withdrawal logic when re-applying, you need to factor in how you're going to update this for live data that has already had applications that have opted in via apply.

Reply:
Post deployment script added